opentelemetry-sdk/src/opentelemetry/sdk/logs/__init__.py

```diff
@@ -155,36 +285,47 @@ def flush(self) -> None:
 
 
 class LogEmitter:
-    # TODO: Add multi_log_processor
     def __init__(
         self,
         resource: Resource,
+        multi_log_processor: Union[
+            SynchronousMultiLogProcessor, ConcurrentMultiLogProcessor
+        ],
         instrumentation_info: InstrumentationInfo,
     ):
         self._resource = resource
+        self._multi_log_processor = multi_log_processor
         self._instrumentation_info = instrumentation_info
 
     @property
     def resource(self):
         return self._resource
 
     def emit(self, record: LogRecord):
-        # TODO: multi_log_processor.emit
-        pass
+        """Emits the :class:`LogData` by associating :class:`LogRecord`
+        and instrumentation info.
+        """
+        log_data = LogData(record, self._instrumentation_info)
+        self._multi_log_processor.emit(log_data)
 
     def flush(self):
-        # TODO: multi_log_processor.force_flush
-        pass
+        """Ensure all logging output has been flushed."""
+        self._multi_log_processor.force_flush()
 
 
 class LogEmitterProvider:
-    # TODO: Add multi_log_processor
     def __init__(
         self,
         resource: Resource = Resource.create(),
         shutdown_on_exit: bool = True,
+        multi_log_processor: Union[
```

Comment:
Does the passed in processor HAVE to be one of the multi-processors?

Reply:
Ideally yes, This will allow users to register multiple log processors (probably they want to export to multiple n/w destinations).